documentation/src/main/wiki/HeaderModifier.md

```diff
@@ -14,6 +14,7 @@ In order to use this modifier it must be declared before the open module in the
 | --------- | ----- | ----------- | --------- |
 | `key` | x | Key for the header | yes |
 | `value` | y | Value for the header | yes |
+| `override` | `true`/`false` | Replace existing header instead of adding new one | no (default false) |
```

Comment:
What's the case of using the Header Modifier without `override=true`?
Is that the case, that the user wants to have more than one header's value?
If that's the case, `override=true` should be the default option and eventually let the user add more header values using `override=false`.
Could you please also make it more clear, in the docs, that `override=false` means, that if the header already occurs in the request, it will not be modified (like with `User-Agent` example, where Chrome instance sets it).

Reply:
Override false is use when you want add new header, true option is used to change headers that are add by Chrome instance. This is why false is default strategy.

I'll make documentation more clear about that.